src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Statistics/BoundaryLevelUpdateRequest.cs

```diff
@@ -0,0 +1,8 @@
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+
+public class BoundaryLevelUpdateRequest
+{
+    public int Id { get; set; }
```

Comment:
The type here should match the type of `BoundaryLevel.Id` so `long` rather than `int`.